### ISSUE TITLE
Missing Icons

### ISSUE THREAD

**Issue by thomascrisan** [Betterbird/thunderbird-patches#107]
Version 102.5.1-bb25 running on Arch Linux in English.  Some of the icons are missing from the left side vertical selection bar.  See attachments.

These icons are present in Thunderbird 102.5.1 (64-bit).

![Selection_017](https://user-images.githubusercontent.com/26765643/205168030-8f5fb102-f3cf-4c30-8fb9-287823c9788f.png)
![Selection_018](https://user-images.githubusercontent.com/26765643/205169327-5f185c8d-6eb1-4939-87cd-81e8c9c4697d.png)

**Comment by Betterbird** [comment]
No idea what that bar is, the spaces toolbar within BB looks fine:
![image](https://user-images.githubusercontent.com/86832723/205172739-96f657f2-9466-4a31-bf11-630b2f67c238.png)

@posi81, when you build 102.5.1 for Arch, can you please take a look.

@thomascrisan: BB seems to be correct, nothing we can do here. If the Arch package isn't correct, please file an issue there.

**Comment by thomascrisan** [comment]
I simply downloaded the archive https://www.betterbird.eu/downloads/LinuxArchive/betterbird-102.5.1-bb25.en-US.linux-x86_64.tar.bz2, extracted it to my /opt directory and executed it.  

When upgrading, I had a number of tabs open in Betterbird in 102.5.0.  When upgrading to 102.5.1 and running Betterbird, the icons were missing.  I closed all of my open tabs, restarted Betterbird, and the problem is gone and I cannot repeat it.

Agreed, close this out.
